shell/models/management.cattle.io.roletemplate.js

```
@@ -185,7 +185,16 @@ export default class RoleTemplate extends SteveDescriptionModel {
   async save() {
     const norman = await this.norman;
 
-    return norman.save();
+    for (const rule of norman.rules) {
+      if (rule.nonResourceURLs.length) {
+        delete rule.resources;
+        delete rule.apiGroups;
+      } else {
+        delete rule.nonResourceURLs;
+      }
+    }
+
+    norman.save();
```

as it doesn't return the result, any failures are ignored